### DISCUSSION TITLE
Welcome to Video-View-Homes-And-Rentals Discussions!

### DISCUSSION THREAD

**Discussion by lonells** [lonells/Video-View-Homes-And-Rentals#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lonells** [comment]
Hello Im Lonnie. I am Rachels assistant from Video View Homes and Rentals nice to meet eveyrone.
I am the acting Project manager until further notice.
here is the url to the existing website < videoviewhomesandrentals.com >.
I will also add you to the Host gator C-panel that is the hosting Platform for the video view homes and rentals website

**Reply by amandudi** [reply]
Hello Lonnie.
I am Jaskaran, currently working on a points 7 to 11. 
About the git repository, have you linked up the repository to hostgator cPanel yet ?

I was wondering to work via FTP or GitHub.

Thanks
